src/util.rs

```diff
@@ -140,11 +140,15 @@ where
                 continue;
             }
         };
-        let t = root.entry(k).as_table_mut().expect("Just checked. qed");
+        let t = root
+            .get_mut(k)
+            .expect("Exists. qed")
+            .as_table_mut()
+            .expect("Is table. qed");
 
         for key in keys {
-            let (name, action) = match t.entry(&key) {
-                Item::Value(Value::InlineTable(info)) => {
+            let (name, action) = match t.get_mut(&key) {
```

Comment:
The `toml_edit` got bumped, since the `entry` API graduated from an actual `get`/`get_mut` like API to a true `entry`-style API. This was a search replace effort to be able to bump the ancient deps.